stores/localeStore.ts

```diff
@@ -9,11 +9,16 @@ export type LocaleDisplay = {
 }
 
 export const useLocaleStore = defineStore('locale', () => {
-    const enUsLocale = localeDisplayOptions.find(o => o.code == Locale.EnUs) as LocaleDisplay
+    const enUsLocale = localeDisplayOptions.find(o => o.code === Locale.EnUs) as LocaleDisplay
     const locale: Ref<LocaleDisplay> = ref(enUsLocale)
 
     function setLocale(selectedLocale: Locale) {
-        locale.value = localeDisplayOptions.find(l => l.code == selectedLocale) as LocaleDisplay
+        locale.value = localeDisplayOptions.find(l => l.code === selectedLocale) as LocaleDisplay
+    }
+
+    function formatLanguageCodeToSimpleText(selectedLocale: string) {
```

Comment:
we should call this `displayText` or something that better describes when to use it. Maybe it's just me, but I wouldn't know what simple vs complex text is

Reply:
Nuxt compiles the i18n so within the store I still have to figure out how the store can have access to translatable strings